### PR TITLE
Model bias analysis: compare the different treatment model families.

### PR DESCRIPTION
**Pull request by jetpack**
Sorry, another huge ipynb diff...

### CODE REVIEW

File: src/Model bias analysis.ipynb [hunk 1]
```diff
@@ -1504,477 +2504,3954 @@
...
-       "      dtype='object')"
-      ]
-     },
-     "execution_count": 138,
-     "metadata": {},
-     "output_type": "execute_result"
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "wiki_cnn_v3_100 EER: 0.74\n",
```
Comment (nthain) [inline src/Model bias analysis.ipynb]: Could you change the print message so it is a bit clearer what is happening here. It seems like the same model (e.g. wiki_cnn_v3_100) is being computed multiple times and getting different EERs. However, I think this is due to different terms being looked at? But do we want to compute the EERs for each model for each term? Or just one EER per model across the entire madlibs dataset?

File: src/Model bias analysis.ipynb [hunk 2]
```diff
@@ -1504,477 +2504,3954 @@
...
+      "wiki_debias_random_cnn_v3_106 EER: 0.28\n",
+      "wiki_debias_random_cnn_v3_107 EER: 0.35\n",
+      "wiki_debias_random_cnn_v3_108 EER: 0.35\n",
+      "wiki_debias_random_cnn_v3_109 EER: 0.36\n",
+      "wiki_debias_random_cnn_v3_110 EER: 0.37\n",
+      "wiki_debias_random_cnn_v3_111 EER: 0.20\n",
+      "wiki_debias_random_cnn_v3_112 EER: 0.28\n",
+      "wiki_debias_random_cnn_v3_113 EER: 0.41\n",
+      "wiki_debias_random_cnn_v3_114 EER: 0.33\n",
+      "wiki
```
Comment (nthain) [inline src/Model bias analysis.ipynb]: I think so.

Comment (nthain) [reply src/Model bias analysis.ipynb]: Oh - this looks misplaced. The I think so is referring to your comment:

"### Weird, is this correct? the TNRs at EER look pretty comparable...
### should we be computing the EER for the overall dataset as opposed to the term subsets?"

File: src/Model bias analysis.ipynb (hunk 1 above)
Comment (jetpack) [reply src/Model bias analysis.ipynb]: Yes, we want the latter. I'm changing that.

**Comment by jetpack** [comment]
Sorry, I know it's a pain to review changes to big notebooks. I think the real solution is to move all non-trivial code to python modules and only use the notebook to display the results. I think it's easy enough to do that as the next step before doing further development.

**Comment by nthain** [comment]
LGTM

**Pull request merged**